### ISSUE TITLE
TFLite Converter not able to convert tf.keras model in TensorFlow 2.0

### ISSUE THREAD

**Issue by margaretmz** [tensorflow/tensorflow#25575]
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>

**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): macOS
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: N/A
- TensorFlow installed from (source or binary):binary
- TensorFlow version (use command below): 2.0.0-dev20190206
- Python version: 3.6
- Bazel version (if compiling from source): N/A
- GCC/Compiler version (if compiling from source): N/A
- CUDA/cuDNN version: N/A
- GPU model and memory: N/A

You can collect some of this information using our environment capture [script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh)
You can also obtain the TensorFlow version with
python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"

**Describe the current behavior**
With TF 2.0 preview, the converter is not able to convert a tf.keras model to tflite either through python or command line

**Describe the expected behavior**
The TFlite converter converts the tf.keras model to tflite model.

**Code to reproduce the issue**
Provide a reproducible test case that is the bare minimum necessary to generate the problem.
Please use this code for your testing:
https://github.com/margaretmz/test-tensorflow-2.0/blob/master/test-tflite-converter/test_tf2.0_keras_to_tflite.py
It contains code that you can use for testing - the conversion works for tf 1.11.0 and tf.12.0 but not for tf 2.0

**Other info / logs**
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.

**Comment by SanthoshRajendiran** [comment]
In tf2.0, there is no direct function as from_keras_model, you have to create a concrete function. 
It works, but on conversion to tflite, other issues are actually coming like some key error for batch normalization as it is falling back to v1 for batch normalization, and the resource nodes for all the operators are getting added up along with this.. I am using TF2.0 alpha release

**Comment by ymodak** [comment]
See https://www.tensorflow.org/lite/r2/convert/concrete_function to know more.
Feel free to reopen issue if running into issues. Thanks!
